_episodes/10-diagnostics.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Writing your own diagnostic script"
+teaching: TBD
+exercises: TBD
+
+questions:
+- "How to read preprocessor output from a Python diagnsotics?"
+- "How to write a new diagnostic in ESMValTool?"
+
+objectives:
+- "Understanding the interface between the ESMValCore preprocessor and a diagnostic script."
+- "Writing a new Python diagnostic"
+
+keypoints:
+- "Take home message 1"
+- "etc ..."
+---
+
+## Introduction
+
+The diagnostic script is an important component of ESMValTool where the actual 
+scientific analysis or performance metric is implemented. With ESMValTool, you 
+can reuse an existing diagnostic, adapt and existing one for your needs or
+write your own new diagnostic.  Diagnostics can be written in one or more open 
+source languages such as Python, R, Julia or NCL but we will focus on understanding 
+and writing Python diagnostics in this lesson. In order to access existing diagnostics or
+to write your own, please install ESMValTool in the development mode on your 
+machine using the instructions from [here](https://esmvalgroup.github.io/ESMValTool_Tutorial/08-development-setup/index.html).
```

Comment:
```suggestion
machine using the instructions from [this episode](08-development-setup/index.html).
```

Reply:
If it works a relative link would be better, I think.

Reply:
also added a hint that the information is in this tutorial rather than an external site.